packages/components/src/table/index.js

```diff
@@ -236,13 +237,14 @@ class TableCard extends Component {
 			rowHeader,
 			rowsPerPage,
 			searchBy,
-			searchParam,
 			showMenu,
 			summary,
 			title,
 			totalRows,
 		} = this.props;
 		const { selectedRows, showCols } = this.state;
+		const searchedValues = query.search ? query.search.split( ',' ) : [];
```

Comment:
do we want to trim away white space or check length of `query.search` here too?

Reply:
That's a good idea! I think trimming the input of the `<Search>` box would the best solution here, so it will also affect the filters search boxes. Created an issue for that: #1437.